### ISSUE TITLE
config clarity - "nosellatloss"

### ISSUE THREAD

**Issue by iceryan354** [whittlem/pycryptobot#23]
In the configuration, "nosellatloss" is the variable with option 0 (false) or 1 (true)
It is described in documentation with double negative:
 "* No not sell at a loss (no small losses but could prevent getting out before a big crash)"
And when the app runs, it is described as:
"Sell At Loss : False"  (with 1)

It is confusing.
nosellatloss = false = sell at loss = false ?

Maybe we can use the same term in both config and app? ("sellatloss" or "nosellatloss" but not both) - my vote is to use "sellatloss" so to avoid double negative....

Or edit the text in app to say "Do not sell at loss"

**Comment by whittlem** [comment]
Thanks, I see how that can be confusing. I will update it as I like your suggestion.

"nosellatloss" set to 1 means do not sell at a loss. 0 means allow the bot to sell if a sell signal is detected.

**Comment by whittlem** [comment]
Fixed
